plugins/inputs/ceph/README.md

```diff
@@ -168,32 +168,52 @@ All admin measurements will have the following tags:
 
 ## Cluster
 
+- ceph_fsmap
+  - fields:
+    - up (float)
+    - in (float)
+    - max (float)
+    - up_standby (float)
+
 - ceph_health
   - fields:
-    - status
-    - overall_status
+    - status (string)
+    - status_code (int)
```

Comment:
Be careful, we try to maintain backward compatibility until major release. I would expect `overall_status` to still be there, but it could be disabled with a feature flag-like setting.